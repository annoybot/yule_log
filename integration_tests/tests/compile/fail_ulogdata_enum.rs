use yule_log::ULogData;

#[derive(ULogData)]
pub enum NotAStruct {
    A,
    B,
}

fn main() {}
