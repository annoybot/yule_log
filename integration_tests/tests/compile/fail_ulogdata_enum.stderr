error: ULogData derive on structs only
 --> tests/compile/fail_ulogdata_enum.rs:4:10
  |
4 | pub enum NotAStruct {
  |          ^^^^^^^^^^
